src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -487,18 +488,19 @@ def count_performance_issues(trace_id: str, params: Mapping[str, str]) -> int:
 @sentry_sdk.tracing.trace
 def create_transaction_params(
     trace_id: str,
-    params: Mapping[str, str],
-) -> Mapping[str, str]:
+    snuba_params: SnubaParams,
+) -> SnubaParams:
     """Can't use the transaction params for errors since traces can be errors only"""
-    transaction_params = params.copy()
+    transaction_params = snuba_params.copy()
```

Comment:
Nit: This copy doesn't need to happen until almost the end of this function as there's several early escapes in this function

Reply:
oh good catch